### ISSUE TITLE
Build issues

### ISSUE THREAD

**Issue by mhaseeb31** [Eigenstate/vmd-python#44]
Hi,
I am trying to build vmd-python, i am getting following error.
Can someone help me resolving this.

Moreover is vmd-python not available as a conda package anymore?

![image](https://user-images.githubusercontent.com/31090998/199305622-96456d7d-8c47-4d6a-92e6-24330bc16612.png)

**Comment by gvlidiya** [comment]
I have the same problem with the build, and conda doesn't find the package.
```
conda install -c "conda-forge/label/cf202003" vmd-python
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
```

**Comment by Eigenstate** [comment]
Fixed in 3.1.1 - conda packages coming soon
